src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantOverlapAnnotator.java

```diff
@@ -90,17 +88,17 @@ public static VariantContext annotateRsID(final List<VariantContext> vcsAtLoc, f
     }
 
     /**
-     * Add overlap attributes to vcToAnnotate against all overlapBindings in tracker
+     * Add overlap attributes to vcToAnnotate against all overlapBindings in featureContext
      *
      * @see #annotateOverlap(java.util.List, String, htsjdk.variant.variantcontext.VariantContext)
      * for more information
      *
-     * @param tracker non-null tracker, which we will use to update the rsID of vcToAnnotate
+     * @param featureContext non-null featureContext, which we will use to update the rsID of vcToAnnotate
      *                for VariantContexts bound to dbSNPBinding that start at vcToAnnotate
```

Comment:
These comments (here and above) imply that the `FeatureContext` is meant to be backed by DNSNP inputs only? Is this correct (based on the code, it isn't)?  "for VariantContexts bound to dbSNPBinding" is a meaningless phrase in GATK4.